### ISSUE TITLE
BLE.scan( ) doesn't work

### ISSUE THREAD

**Issue by teepad** [don/cordova-plugin-ble-central#770]
I'm trying use ble.scan, but I'm not getting any signal. Othes functions is working, like BLE.connect and BLE.enable

![image](https://user-images.githubusercontent.com/60747674/83990785-2354db00-a921-11ea-8470-f6fc736f5a1b.png)

**Comment by xdvarpunen** [comment]
I happened to have same issue. Calling ble instance methods in set interval somehow don't work.

Don't do ble instance method calls in the setInterval function.

**Comment by xdvarpunen** [comment]
Use it more like this:

```javascript
constructor(ble){
    // 15 seconds is too little, put maybe 1min and stop from button?
    ble.scan([], 15, function(device) {
        console.log(JSON.stringify(device));
        ble.connect(device.id, function connectCallback(){
            // do stuff with connected device
        }, function disconnectCallback(){});
    }, failure);
}
```

**Comment by don** [comment]
Don't use setInterval. This might be related to Android 10 permissions too. Fixed in v1.3.0.
